data/curated/border-patrol/meta.yaml

```diff
@@ -0,0 +1,19 @@
+title: U.S. Customs and Border Protection (CBP) encounter data
+article:
+  title: U.S. Border Patrol Encounters
+  url: https://posit.cloud/content/9130042
+data_source:
+  title: U.S. Customs and Border Protection
+  url: https://www.cbp.gov/
+images:
+# Please include at least one image, and up to three images
+- file: inadmissible-aliens-plot.png
+  alt: >
+    A bar plot of monthly inadmissible CHNV (citizens of Cuba, Haiti, Nicaragua, and Venezuela) aliens encountered by U.S. Border Patrol agents from 2020 to 2024.
```

Comment:
Add a source (you?). Also describe the plot content. What does it show? This guide doesn't 100% apply to plots, but it's a good place to start: https://bsky.app/profile/ind3fatigable.xyz/post/3lb5weqbnt22t
A tip I heard that stuck with me is that alt text should serve as a *replacement* for the image, not only a *description* of the image.

Reply:
Did you create the plot? I see you added a credit for the data, but I meant a credit for the image. Note: Looking at past examples won't necessarily show a great track record about this, but it's something I want us to get better about!